App manager causes crash

Im using a rooted OnePlus 3 with Android Pie (LineageOS 16) and starting the app manager causes amaze to crash with the following error:

```
FATAL EXCEPTION: ModernAsyncTask #1
Process: com.amaze.filemanager, PID: 27309
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.support.v4.content.ModernAsyncTask$3.done(ModernAsyncTask.java:161)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.NullPointerException
	at java.io.File.<init>(File.java:283)
	at com.amaze.filemanager.asynchronous.loaders.AppListLoader.loadInBackground(AppListLoader.java:58)
	at com.amaze.filemanager.asynchronous.loaders.AppListLoader.loadInBackground(AppListLoader.java:26)
	at android.support.v4.content.AsyncTaskLoader.onLoadInBackground(AsyncTaskLoader.java:306)
	at android.support.v4.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:59)
	at android.support.v4.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:47)
	at android.support.v4.content.ModernAsyncTask$2.call(ModernAsyncTask.java:138)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 more
```

--- a/app/src/main/java/com/amaze/filemanager/asynchronous/loaders/AppListLoader.java
+++ b/app/src/main/java/com/amaze/filemanager/asynchronous/loaders/AppListLoader.java
@@ -75,6 +75,9 @@ public class AppListLoader extends AsyncTaskLoader<AppListLoader.AppsDataPair> {
     mApps = new AppsDataPair(new ArrayList<>(apps.size()), new ArrayList<>(apps.size()));
 
     for (ApplicationInfo object : apps) {
+      if (object.sourceDir == null) {
+        continue;
+      }
       File sourceDir = new File(object.sourceDir);
 
       String label = object.loadLabel(packageManager).toString();

--- a/build.gradle
+++ b/build.gradle
@@ -32,10 +32,9 @@ allprojects {
         mavenCentral()
     }
     tasks.withType(Test) {
-        maxParallelForks = 4
+        maxParallelForks = 2
         maxHeapSize = "4g"
-        forkEvery = 4
-    }
+        forkEvery = 2   }
 }
 
 spotless {

--- a/build.gradle
+++ b/build.gradle
@@ -32,9 +32,11 @@ allprojects {
         mavenCentral()
     }
     tasks.withType(Test) {
-        maxParallelForks = 2
+        maxParallelForks = 4
         maxHeapSize = "4g"
-        forkEvery = 2   }
+        forkEvery = 4
+        failFast = true
+    }
 }
 
 spotless {

